### PR TITLE
Add pindel

### PR DESCRIPTION
**Pull request by marrip**
add rules to generate pindel config plus script, to run pindel and to convert output to vcf.

### CODE REVIEW

File: workflow/rules/pindel.smk [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        bam="alignment/merge_bam/{sample}_T.bam",
+        bai="alignment/merge_bam/{sample}_T.bam.bai",
+        ref=config["reference"]["fasta"],
+    output:
+        pindel=temp(
+            expand(
+                "cnv_sv/pindel/{{sample}}_{ext}",
+                ext=[
+                    "BP",
+                    "CloseEndMapped",
```
Comment (Smeds) [inline workflow/rules/pindel.smk:19]: are these all output files from pindel?

File: workflow/scripts/generate_pindel_config.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```
Comment (Smeds) [inline workflow/scripts/generate_pindel_config.py:1]: I have added feature request for adding logging and tests to this script

File: workflow/rules/pindel.smk (hunk 1 above)
Comment (marrip) [reply workflow/rules/pindel.smk:19]: yes. Not sure if I forgot any 🤔

File: workflow/scripts/generate_pindel_config.py (hunk 2 above)
Comment (marrip) [reply workflow/scripts/generate_pindel_config.py:1]: I am working on the tests, logging will be added as well 👍

File: workflow/rules/pindel2vcf.smk [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                "INV",
+                "LI",
+                "RP",
+                "SI",
+                "TD",
+            ],
+        ),
+        ref=config["reference"]["fasta"],
+    output:
+        vcf="cnv_sv/pindel/{sample}.vcf",
```
Comment (jonca79) [inline workflow/rules/pindel2vcf.smk:25]: temp

Comment (marrip) [reply workflow/rules/pindel2vcf.smk]: done

**Review by jonca79 — APPROVED** [review]

**Pull request merged**